e2e-environment/Dockerfile

```diff
@@ -32,7 +35,8 @@ ARG CONTRACTS_VENV
 ARG CONTRACTS_PACKAGE_VERSION
 
 RUN python -m venv ${CONTRACTS_VENV}
-RUN . ${CONTRACTS_VENV}/bin/activate && pip install -U pip wheel && pip install raiden-contracts==${CONTRACTS_PACKAGE_VERSION}
+RUN ${CONTRACTS_VENV}/bin/pip install --upgrade pip wheel
+RUN ${CONTRACTS_VENV}/bin/pip install git+https://github.com/weilbith/raiden-contracts@${CONTRACTS_PACKAGE_VERSION}
```

Comment:
Why are we getting it from your fork instead of the main repository?

Reply:
At this point in time I did not want to spam the `origin` repos with such experimental tags. And as it will not stay like that anyway and will change, I thought that is fine.
Do you want me to change it? 🙃

Reply:
As an effect, we'll have an `origin` repo pointing to a fork, which I honestly believe is worse. I'd prefer to have this pointing to a branch, but on a main repo.

Reply:
Sure. I'll do.

Reply:
✔️